daemon/mgr/container.go

```diff
@@ -198,6 +199,8 @@ type ContainerManager struct {
 
 	// eventsService is used to publish events generated by pouchd
 	eventsService *events.Events
+
+	machineMemory uint64
```

Comment:
put this field into ContainerManager seems like not a good idea?

Reply:
Agree. But currently I cannot find a better place for this.  Can you give me some hints? Or we need to calculate the machine memory every time.

Reply:
how about to put it to `ContainerManager.Config`? wdyt?

Reply:
Thanks. I have updated my code. PTAL